### ISSUE TITLE
Can't setup Views

### ISSUE THREAD

**Issue by DynamiteGoesBoom** [dukt/analytics#33]
### Description

I'm not sure what's happening here, and maybe I'm just missing something simple, but I can connect GA to my delegated account for my client. But I cannot setup any views. When I attempt to do so I get this error:

![Screen Shot 2021-02-14 at 3 19 55 PM](https://user-images.githubusercontent.com/39201467/107890851-70d60e80-6ed8-11eb-91f3-fd795099f7fb.png)

Inspecting the `console` I get a 500 error to this file:

`/index.php/admin/actions/analytics/settings/get-account-explorer-data`

Is there some sort of access that I need in order to do so? I do have `ALLOW_UPDATES` set to `false` on the `production` site but am temporarily `ALLOW_ADMIN_CHANGES` set to true so that I can connect Analytics with the website.

Is there something that I'm missing here? Should I try disabling the Cache in my `config/analytics.php` file or something?

### Steps to reproduce

1. Install plugin, 
2. Create a `config/analytics.php` file 
3. Add values for `oauthClientId` and `oauthClientSecret` and confirmed those are correct
4. Attempt to setup a View

### Additional info

- Plugin version: 4.0.11
- Craft version: Craft Pro 3.5.17.1
- PHP version: 7.4.13
- Database driver & version:
- Plugins & versions:

![Screen Shot 2021-02-14 at 3 29 21 PM](https://user-images.githubusercontent.com/39201467/107891041-7122d980-6ed9-11eb-8b59-c89d3122a2e0.png)

I am also not seeing anything in the log files either.

Any help here would be much appreciated!

**Comment by benjamindavid** [comment]
Hey Mark, did you manage to go through the OAuth process properly and connect to Google Analytics on the production website? I'd be curious to see the log files if you can share them with me, they might help understand what's going on if though they don't contain errors. Thank you!

**Comment by DynamiteGoesBoom** [comment]
@benjamindavid thanks for the reply! I don't see anything helpful in the error logs, Here's what's outputting in the PHP logs:

```
[14-Feb-2021 06:25:02] NOTICE: error log file re-opened
[14-Feb-2021 22:45:37] NOTICE: Reloading in progress ...
[14-Feb-2021 22:45:37] NOTICE: reloading: execvp("/usr/sbin/php-fpm7.4", {"/usr/sbin/php-fpm7.4", "--nodaemonize", "--fpm-config", "/etc/php/7.4/fpm/php-fpm.conf"})
[14-Feb-2021 22:45:37] NOTICE: using inherited socket fd=7, "/run/php/php7.4-fpm.sock"
[14-Feb-2021 22:45:37] NOTICE: using inherited socket fd=7, "/run/php/php7.4-fpm.sock"
[14-Feb-2021 22:45:37] NOTICE: fpm is running, pid 846
[14-Feb-2021 22:45:37] NOTICE: ready to handle connections
[14-Feb-2021 22:45:37] NOTICE: systemd monitor interval set to 10000ms

```

I also looked through the other Forge logs, but didn't see anything useful there either.

As I mentioned in my initial comment I'm getting a `500` response from this too:

![Screen Shot 2021-02-17 at 2 41 01 PM](https://user-images.githubusercontent.com/39201467/108271637-54331400-712e-11eb-89a0-d6729dbeda31.png)

Since creating this issue I also tried removing the analytics.php file from config in order to test if the default settings were better suited. I can restore that and try again if you think that could be helpful.

Thanks again!

**Comment by DynamiteGoesBoom** [comment]
I just restored that file and that made no difference.

**Comment by benjamindavid** [comment]
Thanks a lot for the feedback Mark. I know they shouldn’t contain much but could you please your [Craft logs](https://craftcms.com/knowledge-base/locating-error-logs-and-database-backups#error-logs) anyway? Thank you!

**Comment by DynamiteGoesBoom** [comment]
@benjamindavid where should I send those to? I don't think GitHub is the best place for those :)

**Comment by DynamiteGoesBoom** [comment]
I'll email them to you now :) Thanks for the help!

**Comment by benjamindavid** [comment]
Thanks for the logs! I've just checked them and it looks like there might be an issue with your Google Analytics configuration:
> Google Analytics API has not been used in project XXXXXXXXXXX before or it is disabled.

Can you please go to your Google Console and make sure that the Analytics API is configured for the account you are trying to use it with?

> In API Manager → Library, search for “analytics” and enable the “Analytics API”.

Hope this helps!

**Comment by DynamiteGoesBoom** [comment]
@benjamindavid doh, I certainly missed that last step 🤦‍♂️.

Does your plugin work with the latest Craft 3.6.x release? I heard from a colleague that it didn't and just wanted to check with you on that before upgrading this site.

Thanks again!

**Comment by benjamindavid** [comment]
Yeah, Analytics was [having issues](https://github.com/dukt/analytics/issues/34#issuecomment-790544282) with Craft 3.6, but this should now be fixed in [Analytics 4.0.12](https://github.com/dukt/analytics/blob/master/CHANGELOG.md#4012---2021-03-04).
